sql/telemetry_derived/addon_names_v1/query.sql

```diff
@@ -0,0 +1,31 @@
+WITH grouped AS (
+  SELECT
+    addons.key AS addon_id,
+    addons.value.name AS addon_name,
+    COUNT(*) AS occurrences
```

Comment:
This should probably be:
```suggestion
    COUNT(DISTINCT client_id) AS occurrences
```